backend/minors/AAS.yaml

```diff
@@ -28,27 +28,31 @@ req_list:
   requirement.
   double_counting_allowed: false
   max_common_with_major: 1 
-  pdfs_allowed: 1 # only one can count towards the minor
+  pdfs_allowed: 1
   course_list:
   - AAS 244
   - AAS 245
   - AAS 253
   - AAS 254
   - AAS 267
   - AAS 268
+  - AAS 353
+  - AAS 359
+  - AAS 359
```

Comment:
From these OR cases, you have to break them down into subreqs. For reference, check out [AAS.yaml](https://github.com/HoagieClub/plan/blob/6f6ee088b680b8ade0c5f65bd39dc30bd22dd764/backend/certificates/AAS.yaml#L4)

Reply:
oh they have the exact same thing